### ISSUE TITLE
[Enhancement] Option to specify mode to use [light, dark]

### ISSUE THREAD

**Issue by danieltomasz** [mdelobelle/obsidian_supercharged_links#149]
Would be great to have an option to specify the mode when the css is applied, 
Currently I use great  pastel colors, it looks great with dark mode,  but the titles are illegible when in the light mode

Solution would be to have optional setting to specify the condition when the formatting is applied (light, dark, both)

**Comment by HEmile** [comment]
You can achieve this with CSS for sure. Do you mean in the generator? I think that could be a good idea.

**Comment by danieltomasz** [comment]
Yes, I mean in the generator :)  I am thinking how this might be solved from UI/UX point of view, to not burden user :) 

Maybe in the the "Edit selector" field adding toggle "different styles for light/dark theme" will cause adding option for colors for light and dark mode in "Style Settings" for this specific tag

this is not crucial but nice to have feature for completness

**Comment by HEmile** [comment]
Yes, I like the idea! I'll add it when I next iterate the plugin.
